python/cuspatial/cuspatial/core/_column/geocolumn.py

```diff
@@ -11,6 +12,17 @@
 from cuspatial.core._column.geometa import Feature_Enum, GeoMeta
 from cuspatial.utils.column_utils import empty_geometry_column
 
+
+class ColumnType(Enum):
+    """Types which are returned by GeoColumn.typeclass."""
```

Comment:
`GeoColumn.typeclass` doesn't seem like added. Is this meant to be added?

Reply:
That's a stale comment. I'll update.

Reply:
I ended up putting all of the `ColumnType` logic into `geoseries.py` because the `contains_only_*` methods all take a `GeoSeries` instead of a column. However, the `ColumnType` Enum remains in `geocolumn.py`. I'm open to refactoring ideas. Maybe I rename it `SeriesType` and move the enum into `geoseries.py`?